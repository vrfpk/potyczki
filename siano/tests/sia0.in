4 4
1 2 4 3
1 1
2 2
3 0
4 4
