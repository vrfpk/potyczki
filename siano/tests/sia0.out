6
6
18
0
